NHLBI_BioData_Catalyst/R/7_Harmonization_with_PICSURE.ipynb

```diff
@@ -80,13 +87,11 @@
    "source": [
     "Sys.setenv(TAR = \"/bin/tar\")\n",
     "options(unzip = \"internal\")\n",
-    "install.packages(\"https://cran.r-project.org/src/contrib/Archive/devtools/devtools_1.13.6.tar.gz\", repos=NULL, type=\"source\")\n",
-    "install.packages(\"https://cran.r-project.org/src/contrib/R6_2.5.1.tar.gz\", repos=NULL, type=\"source\")\n",
-    "install.packages(\"https://cran.r-project.org/src/contrib/hash_2.2.6.1.tar.gz\", repos=NULL, type=\"source\")\n",
-    "install.packages(c(\"urltools\"),repos = \"http://cran.us.r-project.org\")\n",
     "devtools::install_github(\"hms-dbmi/pic-sure-r-client\", force=T)\n",
     "devtools::install_github(\"hms-dbmi/pic-sure-r-adapter-hpds\", force=T)\n",
-    "devtools::install_github(\"hms-dbmi/pic-sure-biodatacatalyst-r-adapter-hpds\", force=T)"
+    "devtools::install_github(\"hms-dbmi/pic-sure-biodatacatalyst-r-adapter-hpds\", force=T)\n",
+    "library(hpds)\n",
+    "library(ggplot2)"
```

Comment:
Needed library(stringr) for Terra.

Reply:
Resolved in [25d486fe997b29c798c6cfdf6800bbd5a97a19f3](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/25d486fe997b29c798c6cfdf6800bbd5a97a19f3)